gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactor.java

```diff
@@ -141,6 +142,13 @@
   public static final String COMPACTION_DATASETS_FINDER = COMPACTION_PREFIX + "datasets.finder";
   public static final String DEFAULT_COMPACTION_DATASETS_FINDER = TimeBasedSubDirDatasetsFinder.class.getName();
 
+
+  // Rename source directories as a compaction complete indication
```

Comment:
Should mention that compaction jobs using this completion mode can't share input sources.